cc-mt-service/src/handleData.ts

```diff
@@ -0,0 +1,108 @@
+import axios from 'axios';
+import dotenv from 'dotenv';
+import qs from 'qs';
+import {AdvertData} from './types';
+
+dotenv.config();
+
+// variables
+const c_id: string = process.env.MT_CLIENTID ?? '';
+const c_secret: string = process.env.MT_CLIENTSECRET ?? '';
+const scope: string = process.env.MT_SCOPE ?? '';
+
+/**
+ * Get token to use test.materiaalitori.fi API to make possible to add adverts
+ * @param client 
+ * @param secret 
+ * @param scope 
+ * @returns token to Materiaalitori API
+ */
+async function getToken(client:string, secret:string, scope:string) {
+    try {
+        const data=qs.stringify({
+            'client_secret':secret,
+            'client_id':client,
+            'grant_type':'client_credentials',
+            'scope':scope
+        });
+        const config={
+            method:'post',
+            url:'https://test-auth.materiaalitori.fi/connect/token?grant_type=client_credentials&scope=' + scope,
+            headers:{
+              'Content-Type':'application/x-www-form-urlencoded'
+            },
+            data:data
+        };
+        const response=await axios(config);
+        const responseStr=response.data.access_token;
+        return responseStr
+    } catch (error) {
+        console.log(error);
+    }
+}
+
+// Posts data to Materiaalitori test API
+export async function postAdvert() {
+    try {
+        // Valid data which can be posted to test.materiaalitori.fi
+        const dataObject:AdvertData = {
```

Comment:
Add the dataObject as a parameter to this postAdvert() function so this can be used to post anything